tsconfig.json

```diff
@@ -24,5 +24,6 @@
         "skipDefaultLibCheck": true, // Skip type checking .d.ts files that are included with TypeScript
         "downlevelIteration": true
     },
-    "exclude": ["node_modules", "build", "**/__tests__/**/*", "**/*.js", "**/*.jsx"] // *** The files to not type check ***
+    "include": ["utils.d.ts"],
```

Comment:
Since `include` has been override in each package `tsconfig` file, This won’t do anything for now and we have to also add `utils.d.ts` in each package. Later when we fix `tsconfig` files in each package to rely on the root one we can remove `include` from each package and this will be used.

Reply:
@iman-fs I believe this is not a TODO since it’s already done. If someone comes here and wonders why they will probably git blame the line and come to this PR and see the comment.

What do you think? 🤔

Reply:
You're right. I thought it's not done.